src/DroneSafetyApi/Controllers/HeatmapsController.cs

```diff
@@ -27,11 +32,18 @@ public IActionResult Get([FromQuery]HeatmapsQuery query)
             {
                 return new BadRequestResult();
             }
-            // TODO: Decide optimal radius value, and put value/computation in appropriate place.
-            var intersectionHazards = DataPoints.GetDataPointsInRadius(query.Centre, 100000);
-            var heatmapsResponse = DataPointsToHeatmaps.ConvertToHeatmapResponse(
-                query.Resolution,
+            var radius = (int) (Math.Sqrt(((query.Area.Max.Latitude - query.Area.Min.Latitude) *
+                (query.Area.Max.Latitude - query.Area.Min.Latitude)) +
```

Comment:
The squaring should be done more nicely. It's okay if you have something like x * x, but when you have something complicated being squared then it should either be encapsulated inside a variable, or you should have a squaring method.